common/src/main/java/whocraft/tardis_refined/common/tardis/TardisArchitectureHandler.java

```diff
@@ -90,21 +90,12 @@ public static void buildAirlockEntranceFromStructure(StructureTemplate template,
     }
 
     public static void generateEssentialCorridors(ServerLevel serverLevel) {
-        // Generate corridor hub
-        Optional<StructureTemplate> structureNBT = serverLevel.getLevel().getStructureManager().get(new ResourceLocation(TardisRefined.MODID, "corridors/corridor_hub_roomless"));
-        structureNBT.ifPresent(structure -> {
-            BlockPos offsetPosition = new BlockPos(13, 28, 5);
-            structure.placeInWorld(serverLevel.getLevel(), CORRIDOR_ENTRY_POS.subtract(offsetPosition), CORRIDOR_ENTRY_POS.subtract(offsetPosition), new StructurePlaceSettings(), serverLevel.getLevel().random, 3);
-        });
-
-        TardisLevelOperator.get(serverLevel).ifPresent(tardisLevelOperator -> generateArsTree(tardisLevelOperator, serverLevel));
-
-        // Generate workshop.
-        structureNBT = serverLevel.getLevel().getStructureManager().get(new ResourceLocation(TardisRefined.MODID, "rooms/workshop"));
-        structureNBT.ifPresent(structure -> {
-            BlockPos position = new BlockPos(977, 99, 9);
-            structure.placeInWorld(serverLevel.getLevel(), position, position, new StructurePlaceSettings(), serverLevel.getLevel().random, Block.UPDATE_ALL);
-        });
+        // To be placed back again later.
```

Comment:
This won't be getting merged with commented code

Reply:
Done but replaced with a message.